### DISCUSSION TITLE
Japanese localization suggestion

### DISCUSSION THREAD

**Discussion by koike-ya** [GitbookIO/community#721] [Feature Requests]
"Ask or Search" button at the top right has two lines of characters in Japanese `質問するか検索\nする`, which is making the UI a little bit unnatural.
I suggest `質問または検索` or `質問や検索`, which are the nouns of "Ask or Search".
I'm Japanese developer so I can assure these suggestions are valid and natural in Japanese.

**Comment by addisonschultz** [comment]
Thanks for pointing this out! I agree with your points. I've opened up a PR to fix this here: https://github.com/GitbookIO/gitbook/pull/2462

**Comment by koike-ya** [comment]
Thank you for the quick action. Very much appreciated!!
